server/server.go

```diff
@@ -293,6 +304,19 @@ func (w *otherWorker) Do(job interface{}) interface{} {
 		w.s.stats.logRequestDuration(requestBegin)
 		return makeRespondResponse(pkt.ID, res)
 
+	case protocol.OpRPC:
+		codec := newServerCodec(pkt.Payload)
+
+		err := w.s.dispatcher.ServeRequest(codec)
+		if err != nil {
+			log.Errorf("Worker %v: ServeRPC: %v", w.name, err)
+			w.s.stats.logInvalid(requestBegin)
+			return makeErrResponse(pkt.ID, protocol.ErrInternal)
+		}
+
+		w.s.stats.logRequestDuration(requestBegin)
+		return makeRespondResponse(pkt.ID, codec.response)
+
```

Comment:
Should we be concerned about latency here? If this runs in the normal worker pool, then blocking RPC requests could hang up the server's ability to do non-ECDSA operations. Are we expecting the RPC server to always be running on localhost or something?